model-test/src/test/java/hu/blackbelt/judo/meta/jsl/runtime/NamedTests.xtend

```diff
@@ -8,25 +8,420 @@ import org.eclipse.xtext.testing.validation.ValidationTestHelper
+             derived Timestamp xTimestamp => `2020-02-18T10:11:12`;
+             derived Time xTime => `12:23:56.1`;
+         }
+         '''.parse => [
+             assertFirstLowSecondUpperNamedError("xBool")
+             assertFirstLowSecondUpperNamedError("xString")
+             assertFirstLowSecondUpperNamedError("xNumeric")
+             assertFirstLowSecondUpperNamedError("xEnum")
+             assertFirstLowSecondUpperNamedError("xDate")
+             assertFirstLowSecondUpperNamedError("xTimestamp")
+             assertFirstLowSecondUpperNamedError("xTime")
+         ]
+    }
+
+        @Test
+        @TestCase("TC021")
+        @Requirement(reqs = #[
+                "REQ-SYNT-001",
+                "REQ-SYNT-002",
+                "REQ-SYNT-003",
+                "REQ-SYNT-004",
+                "REQ-MDL-001",
+                "REQ-MDL-003",
+                "REQ-TYPE-001",
+                "REQ-TYPE-002",
+                "REQ-TYPE-004",
+                "REQ-TYPE-005",
+                "REQ-TYPE-006",
+                "REQ-TYPE-007",
+                "REQ-TYPE-008",
+                "REQ-TYPE-009",
+                "REQ-ENT-001",
+                "REQ-ENT-009",
+                "REQ-ENT-011"
+        ])
+        def void testNamingOfQuery() {
+            '''
+             model modelTC021;
+
+             import judo::types;
+
+             type string SType(min-size = 0, max-size = 10);
+
+             enum Enum {
+                  a0 = 0;
+                  b1 = 1;
+             }
+
+             entity Ent1 {
+                 query Boolean xBool() => true;
+                 query SType xString() => "String";
+                 query Integer xNumeric() => 1;
+                 query Enum xEnum() => Enum#a0;
+                 query Date xDate() => `2023-11-09`;
+                 query Timestamp xTimestamp() => `2020-02-18T10:11:12`;
+                 query Time xTime() => `12:23:56.1`;
+             }
+
+             query Ent1[] xAllEnt1() => Ent1!all();
+             '''.parse => [
+                  assertFirstLowSecondUpperNamedError("xBool")
+                  assertFirstLowSecondUpperNamedError("xString")
+                  assertFirstLowSecondUpperNamedError("xNumeric")
+                  assertFirstLowSecondUpperNamedError("xEnum")
+                  assertFirstLowSecondUpperNamedError("xDate")
+                  assertFirstLowSecondUpperNamedError("xTimestamp")
+                  assertFirstLowSecondUpperNamedError("xTime")
+                  assertFirstLowSecondUpperNamedError("xAllEnt1")
+             ]
+        }
+
+        def private void assertFirstLowSecondUpperNamedError(ModelDeclaration modelDeclaration, String namedElement) {
+            modelDeclaration.assertError(
+                JsldslPackage::eINSTANCE.named,
+                JslDslValidator.JAVA_BEAN_NAMING_ISSUE,
+                "The first character cannot be lowercase and the second character uppercase '"+namedElement+"'."
```

Comment:
Missing spaces

Reply:
character uppercase in

Reply:
https://github.com/BlackBeltTechnology/judo-meta-jsl/pull/140/commits/bfe2c01bd87fdf975f370ad901e05ffd10cb7998